packages/preset-mini/src/_variants/pseudo.ts

```diff
@@ -138,23 +139,39 @@ function taggedPseudoClassMatcher(tag: string, parent: string, combinator: strin
     ]
   }
 
+  const matchPseudoVar = (input: string): [label: string, rest: string, prefix: string] | undefined => {
+    const match = input.match(pseudoVarRE)
+    if (!match)
+      return
+    const [original, fn, pseudoValue] = match
+    const label = match[3] ?? ''
+    const pseudo = `:${fn}(${pseudoValue})`
+
+    return [
+      label,
+      input.slice(original.length),
+      `${parent}${escapeSelector(label)}${pseudo}`,
+    ]
+  }
+
   return {
     name: `pseudo:${tag}`,
     match(input, ctx) {
       if (!(splitRE && pseudoRE && pseudoColonRE)) {
         splitRE = new RegExp(`(?:${ctx.generator.config.separators.join('|')})`)
         pseudoRE = new RegExp(`^${tag}-(?:(?:(${PseudoClassFunctionsStr})-)?(${PseudoClassesStr}))(?:(/\\w+))?(?:${ctx.generator.config.separators.join('|')})`)
```

Comment:
confused me on this, it seems can implement custom arguments at the beginning, but didn't

don't know if there has some edge case that I just add code